### PR TITLE
Updates from bedrock

### PR DESCRIPTION
**Pull request by MozmarRobot**
From file changes in https://github.com/mozilla/bedrock/commit/bedb8ae22add26d76b203fed9c03f9db34b67bb1

### CODE REVIEW

File: en/brands.ftl [hunk 1]
```diff
@@ -18,7 +18,6 @@
 -brand-name-mozilla = Mozilla
 -brand-name-mozilla-corporation = Mozilla Corporation
 -brand-name-mozilla-foundation = Mozilla Foundation
--brand-name-netscape = Netscape
```
Comment (peiying2) [inline en/brands.ftl:21]: We need to keep this name from a different PR.

**Review by peiying2 — CHANGES_REQUESTED** [review]
@alexgibson @craigcook Please keep Netscape in the brands.ftl which was added from a different PR last week. Otherwise, it looks good.

**Comment by alexgibson** [comment]
@peiying2 updated thank you

**Review by peiying2 — APPROVED** [review]
Thanks for the fix. Looks good now.
